docker/pythonpath_dev/superset_config.py

```diff
@@ -106,6 +106,7 @@ class CeleryConfig(object):
 WEBDRIVER_BASEURL = "http://superset:8088/"
 # The base URL for the email report hyperlinks.
 WEBDRIVER_BASEURL_USER_FRIENDLY = WEBDRIVER_BASEURL
+IMPALA_POLL_INTERVAL = int(timedelta(seconds=5).total_seconds())
```

Comment:
I don't think you need this if the value is the same as the config file.

Reply:
I added this parameter for running the docker container environment, and I can remove it if don't need it

Reply:
I have removed the configuration property，thank you